subprojects/docs/src/snippets/tutorial/manifest/tests/manifest.sample.conf

```diff
@@ -1,2 +1,2 @@
 executable: gradle
-args: tasks
+args: fooJar barJar
```

Comment:
Shouldn't we also call `jar` task here as well?

Reply:
Fixed